packages/react-scripts/fixtures/kitchensink/README.md

```diff
@@ -4,10 +4,10 @@ This is an end to end kitchensink test suite, but has multiple usages in it.
 
 ## Running the test suite
 
-Tests are automatically run by the CI tools.  
+Tests are automatically run by the CI tools.
 In order to run them locally, without having to manually install and configure everything, the `yarn e2e:docker` CLI command can be used.
 
-This is a simple script that runs a **Docker** container, where the node version, git branch to clone, test suite, and whether to run it with `yarn` or `npm` can be chosen.  
```

Comment:
Double trailing white spaces are meaningful in markdown. They are converted into a new line